e2e/tests/regression/test_backup.robot

```diff
@@ -116,12 +116,9 @@ Test Uninstallation With Backups
     And Wait for volume 0 healthy
     And Write data 0 to volume 0
 
-    # create failed backups by resetting backupstore and create backups without available backup targets
-    And Reset Backupstore
-    And Create backup 0 for volume 0    wait=False
+    And Create backup 0 for volume 0
+    And Create error backup for volume 0
     And Verify backup list contains errors for volume 0
-    And Set Backupstore
```

Comment:
The original test case intends to test `Test Uninstallation With Backups`. Therefore, the following two steps should be retained:
```
And Set Backupstore
And Create backup 1 for volume 0
```

Reply:
If you didn't `Reset Backupstore`, you don't need to `Set Backupstore` again.

`Create backup 0 for volume 0` is still there.